### PR TITLE
Update bundle_fi.properties

### PR DESCRIPTION
**Pull request by TheJaaju**
+ translations

### CODE REVIEW

File: core/assets/bundles/bundle_fi.properties [hunk 1]
```diff
@@ -234,7 +234,7 @@ off = Pois
 save.autosave = Automaattitallennus: {0}
 save.map = Kartta: {0}
 save.wave = Taso {0}
-save.mode = Gamemode: {0}
+save.mode = Pelimuoto: {0}
 save.date = Viimeksi tallennettu: {0}
```
Comment (valtsu41) [inline core/assets/bundles/bundle_fi.properties]: "Pelitila" was used previously

File: core/assets/bundles/bundle_fi.properties [hunk 2]
```diff
@@ -118,8 +118,8 @@ mod.enable = Käytä
 mod.requiresrestart = Peli suljetaan jotta muutokset voisivat toteutua.
 mod.reloadrequired = [scarlet]Vaatii Uudelleenkäynnistystä
 mod.import = Tuo modi
-mod.import.github = Import GitHub Mod
-mod.item.remove = This item is part of the[accent] '{0}'[] mod. To remove it, uninstall that mod.
+mod.import.github = Tuo GitHub Modi
+mod.item.remove = Tämä esine on osa[accent] '{0}'[] modia. Poistaaksesi sen, Poista tuon modin asennus.
```
Comment (valtsu41) [inline core/assets/bundles/bundle_fi.properties]: Would "Poistaaksesi sen sinun tulee poistaa tuon modin asennus." be better?

**Comment by valtsu41** [comment]
What about the title cases?

**Comment by TheJaaju** [comment]
Where can I find the titles?

**Comment by TheJaaju** [comment]
or did you mean the item names e.g. Large Door(Suuri Ovi) to Large door (Suuri ovi)

**Comment by valtsu41** [comment]
Yes, that would match both the style already used and finnish grammar rules better.

**Comment by valtsu41** [comment]
Great, thanks!

**Pull request merged**